### ISSUE TITLE
how long it took for your model, which was trained on 4 RTX4090，and how many epochs it took to achieve the current level of performance.

### ISSUE THREAD

**Issue by lmx492274842** [TonyXuQAQ/RNGDetPlusPlus#24]
谢谢你提供的代码，我是机器学习方面的小白，我想根据你们的模型重新训练以用在别的领域上，我尝试跑了一个自制的训练集，但效果很微弱（当然这是预料之内的）。我想知道你们用的4张4090训练花了多久？训练了多少轮才达到你们现在的效果？
Thank you for providing the code. I'm relatively new to machine learning, and I'm looking to retrain your model for use in a different domain. I've tried running it on my custom dataset, but the results are quite weak (which was expected, of course). I'm curious to know how long it took for your model, which was trained on 4 RTX4090。 and how many epochs it took to achieve the current level of performance.

**Comment by TonyXuQAQ** [comment]
Hi, sorry for the late reply. Instead of training time/epoch, you should increase the size of the training dataset and make sure the generated label is correct.

**Comment by lmx492274842** [comment]
OK，I have understand. Thank you！
